test/clj/matcher_combinators/matchers_test.clj

```diff
@@ -38,177 +37,177 @@
 (defn invalid-type? [actual]
   (instance? InvalidMatcherType actual))
 
-(fact "in-any-order using matcher ordering with maximum matchings for diff"
-  (c/match (m/in-any-order [a-nested-map b-nested-map])
-    [a-nested-map a-nested-map])
-  => (just {::result/type   :mismatch
-            ::result/value  (just [a-nested-map (contains {:id map? :model mismatch?})]
-                                  :in-any-order)
-            ::result/weight number?}))
-
 (defn one-mismatch? [mismatch-list]
   (= 1 (count (filter #(or (mismatch? %) (missing? %)) mismatch-list))))
 
-(fact "Ensure that in-any-order always prints the match with the fewest
-       number of matchers that don't match"
-  (map #(->> %
-             (c/match (m/in-any-order [1 2 3 4]))
-             ::result/value)
-       (combo/permutations [1 2 3 500]))
-  => (has every? one-mismatch?))
-
-(facts "Show how input ordering affects diff size (when it ideally shouldn't)"
-  (fact "Given a particular input ordering, in-any-order shows the smallest diff"
-    (->> [{:a 2} {:b 2}]
-         (c/match (m/in-any-order [{:a 1} {:a 1 :b 2}]))
-         ::result/value
-         (map vals))
-    => (has every? one-mismatch?))
-
-  (fact "in-any-order minimization doesn't find the match ordering that leads
+(deftest in-any-order
+  (testing "matcher ordering with maximum matchings for diff"
+    (is (match?
+         {::result/type   :mismatch
+          ::result/value  (m/embeds [a-nested-map {:id map? :model mismatch?}])
+          ::result/weight number?}
```

Comment:
if you want a behaviorally equivalent test, you need to wrap this map in an `equals` matcher